source/main.cpp

```diff
@@ -129,7 +129,7 @@ int main(int argc, char **argv) {
 	int patchfile = 0;
 	int header;
 	rawDataOffset=0;
-	char datName[]="YS:/" DATNAME;
+	char datName[MAX_DATNAME_LEN]="YS:/" DATNAME;
```

Comment:
@soarqin All well, then. I wonder if you run a DS game first to clean the profile, and reinstall the new path.. what is the result then?